### PR TITLE
PMM-7658 Release Notes 2.15.1

### PR DESCRIPTION
**Pull request by PaulJacobs-percona**

### CODE REVIEW

File: docs/release-notes/2.15.1.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## Bugs Fixed
+
+* [PMM-7635](https://jira.percona.com/browse/PMM-7635): Investigate grafana-server high CPU consumption
```
Comment (denisok) [inline docs/release-notes/2.15.1.md:25]: could you please align both high CPU together one after another and they rightly should be on top as most critical issue we have this patch release for ?

File: docs/using/alerting.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 *Integrated Alerting* lets you know when certain system events occur.
 
-> <b style="color:goldenrod">Caution</b> Integrated Alerting is a [technical preview](../../../details/glossary.md#technical-preview) and is subject to change.
+> <b style="color:goldenrod">Caution</b> Integrated Alerting is a [technical preview](../details/glossary.md#technical-preview) and is subject to change.
```
Comment (denisok) [inline docs/using/alerting.md:5]: is this for release notes?

Comment (PaulJacobs-percona) [reply docs/using/alerting.md:5]: No, it was a change in my main that got caught up. You can ignore it (just a cosmetic item).

File: docs/release-notes/2.15.1.md (hunk 1 above)
Comment (PaulJacobs-percona) [reply docs/release-notes/2.15.1.md:25]: Will do.

**Review by denisok — APPROVED** [review]

**Review by denisok — APPROVED** [review]

File: docs/release-notes/2.15.1.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## Release Highlights
+
+This patch release fixes performance issues discovered in systems where a majority of Services use push mode, together with other small fixes.
+
+## Bugs Fixed
+
+* [PMM-7635](https://jira.percona.com/browse/PMM-7635), [PMM-7713](https://jira.percona.com/browse/PMM-7713): Fix high CPU and Memory consumption by Victoria Metrics and `grafana-server` after upgrade to 2.15.0 with large numbers of services in 'push' mode.
```
Comment (rnovikovP) [inline docs/release-notes/2.15.1.md:29]:
```suggestion
* [PMM-7635](https://jira.percona.com/browse/PMM-7635): Fix high CPU and Memory consumption by Grafana server after upgrade by docker replacement to 2.15.0 with large numbers of services in 'push' mode.
* [PMM-7713](https://jira.percona.com/browse/PMM-7713): Fix high CPU and Memory consumption by Victoria Metrics after upgrade by docker replacement to 2.15.0 .PMM-7635: Fix high CPU and 
```

**Review by rnovikovP — CHANGES_REQUESTED** [review]
there  2 separate issues, and we need to mention docker  update way

**Review by rnovikovP — APPROVED** [review]

**Pull request merged**